### DISCUSSION TITLE
Could not find ZKsync plugin in the Remix IDE

### DISCUSSION THREAD

**Discussion by radikhal** [Cyfrin/foundry-full-course-cu#2841] [Q&A]
![chrome-capture-2024-11-10](https://github.com/user-attachments/assets/0054f316-3db4-4b60-a420-e52e8a3b7a74)

Hi guys, I could not find the zksync plugin in the remix IDE. How do I find it.

**Comment by EngrPips** [comment]
This could be an internet connection issue or maybe clear your browser data and retry.

**Reply by radikhal** [reply]
Okay thanks

**Reply by EngrPips** [reply]
Anytime, Friend.
